finance-tracker/config/database.yml

```diff
@@ -0,0 +1,27 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
```

Comment:
Why not postgres here as well?

Reply:
because pg is to host on heroku and sqlite3 is more convenient in development. Did the same for alpha blog, will it be better if both are the same?

Reply:
Of course its better to ensure both is the same.
Tell me why

Reply:
So that when migrating to production, there won't be any unforeseen issues due to different databases in development.

Reply:
see? you know. Still ask me.
tsk tsk